superset-frontend/src/views/CRUD/hooks.ts

```diff
@@ -464,7 +467,7 @@ export function useImportResource(
           updateState({ loading: false });
         });
     },
-    [],
+    [handleErrorMsg, resourceLabel, resourceName],
```

Comment:
handleErrorMsg is not memoized upstream, which could cause troubles, even if this is a callback (the performance impact of constant rerenders could yield some issues).
@evans maybe this is a bit out of scope, so up to you, but if we can check the code and see if we can properly memoize the chain here that would be great.

Reply:
Everyone keeps pinging this other guy 🤣 

Appreciate the keen eyes on memoization, however. Generally, it's great to fix perf issues like this as bycatch, but I'm perfectly OK with filing a separate issue. That might help both PRs get through faster, and prevent larger "slippery slope" PRs.

Let's just not lose track of the task... similar memoization (or lack thereof) issues have been cropping up all over, and the gains from fixing them are tangible.